### PR TITLE
feat: Add Pageview Tracker

### PR DESCRIPTION
**Pull request by hismethod**
## Description
Added Page View & Verify Button Tracker

## What type of PR is this?
- [V] 🍕 Feature

### CODE REVIEW

**Review by lukasjhan — COMMENTED** [review]
Is it even necessary to track pressing a button?

**Comment by hismethod** [comment]
> Is it even necessary to track pressing a button?

Not necessary. It's to monitor how often the button is being used.

**Comment by lukasjhan** [comment]
> > Is it even necessary to track pressing a button?
> 
> �Not necessary. It's to monitor how often the button is being used.

Then let's just track only page view :)

**Comment by hismethod** [comment]
> > > Is it even necessary to track pressing a button?
> > 
> > 
> > �Not necessary. It's to monitor how often the button is being used.
> 
> Then let's just track only page view :)

Removed :)

File: src/hooks/ga.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { useLocation } from 'react-router-dom';
+
+const GA_TRACKING_ID = 'G-YSLD6T408C';
+
+function usePageViews() {
+  const location = useLocation();
+
+  useEffect(() => {
+    (window as any).gtag('config', GA_TRACKING_ID, {
+      page_path: location.pathname + location.search,
```
Comment (lukasjhan) [inline src/hooks/ga.hooks.ts]: Does it record the URL query? Maybe later, we'll make the share feature using query.

Comment (hismethod) [reply src/hooks/ga.hooks.ts]: Okay. When site entered it only logs entry paths, not other details like URL queries.

**Review by lukasjhan — APPROVED** [review]

**Pull request merged**